homeassistant/components/light/services.yaml

```diff
@@ -9,15 +9,12 @@ turn_on:
   fields:
     transition:
       name: Transition
-      description: Duration in seconds it takes to get to next state.
-      example: 60
+      description: Duration it takes to get to next state.
       selector:
         number:
           min: 0
           max: 300
-          step: 1
           unit_of_measurement: seconds
-          mode: slider
```

Comment:
Something is off with the default.

According to the schema:

The default is `slider`:

```py
vol.Optional(CONF_MODE, default="slider"): vol.In(["box", "slider"])
```

however, the frontend doesn't seem to care:

![image](https://user-images.githubusercontent.com/195327/118458450-0dd45d80-b6fb-11eb-82fd-165b009a2bcc.png)

Reply:
Hm. The step is ignored, too.

Reply:
I've reached out to frontend to see what is going on.

Reply:
Fixed in home-assistant/frontend#9190